.github/workflows/build.yml

```diff
@@ -0,0 +1,36 @@
+name: Python package
```

Comment:
And rename this too? I don't know what to though

Reply:
Yeah...same. I pushed something, but mostly in fun. :-/ Naming is hard.